bodhi/client/__init__.py

```diff
@@ -98,6 +98,8 @@ def _set_logging_debug(ctx, param, value):
 new_edit_options = [
     click.option('--user'),
     click.option('--password', hide_input=True),
+    click.option('--addbuilds', help='Add Comma-separated list of build nvr'),
```

Comment:
I just realized that this are the shared options between the update new and the update edit command. I think these options are only really useful for edit. So they should probably be added directly to the edit method. What do you think ?

see https://github.com/fedora-infra/bodhi/blob/develop/bodhi/client/__init__.py#L440 for an example.

Reply:
I agree.